modules/ROOT/pages/connected-apps-devs-end-users.adoc

```diff
@@ -0,0 +1,20 @@
+= Connected Apps for Developers and End Users
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+== Developers
+
+Third-party developers who have their own websites or web apps can allow users to log in using the *Log in with Anypoint Platform* option. For seamless implementation, there is a standard button that can be integrated into the front end of your application with a pre-built template and set of components. For more information about how to embed the button, refer to the  https://github.com/anypoint-web-components/anypoint-signin/tree/master[developer tutorial] and the http://anypoint-signin-test.surge.sh/[developer playground]. 
```

Comment:
Where did this surge.sh website come from? I'm concerned about endorsing it. It doesn't even support https